x-pack/legacy/plugins/siem/public/components/formatted_date/index.tsx

```diff
@@ -62,3 +63,36 @@ export const FormattedDate = pure<{
 );
 
 FormattedDate.displayName = 'FormattedDate';
+
+/**
+ * Renders the specified date value according to under/over one hour
+ * Under an hour = relative format
+ * Over an hour = in a format determined by the user's preferences,
+ * with a tooltip that renders:
+ * - the name of the field
+ * - a humanized relative date (e.g. 16 minutes ago)
+ * - a long representation of the date that includes the day of the week (e.g. Thursday, March 21, 2019 6:47pm)
+ * - the raw date value (e.g. 2019-03-22T00:47:46Z)
+ */
+
+export const FormattedRelativePreferenceDate = ({ value }: { value?: string | number | null }) => {
+  if (value == null) {
+    return getOrEmptyTagFromValue(value);
+  }
+  const DATE = getMaybeDate(value);
+  const TO = moment(new Date());
+  const FROM = TO.clone().subtract(1, 'hours');
+  const date = DATE.toDate();
+
+  return DATE.isValid() ? (
+    <LocalizedDateTooltip date={date}>
+      {DATE.isBetween(FROM, TO) ? (
```

Comment:
Just mentioning as there are always a ton of _fun_ ways to do date arithmetic with moment. 😉 The following should work as well (and requires a couple less `const`'s), but you would also need to swap the components as the truthy value is for when it is within the relative date range:

``` ts
moment(date).add(1, 'hours').isBefore(new Date())
```